File: programmers.co.kr/hash/1.py
# https://programmers.co.kr/learn/courses/30/lessons/42576?language=python3
# Unfinished Marathoner

import collections


def solution(participant, completion):
    a = collections.Counter(participant) - collections.Counter(completion)
    answer = list(a.keys())[0]

    return answer

File: programmers.co.kr/hash/1.cpp
// https://programmers.co.kr/learn/courses/30/lessons/42576

#include <string>
#include <unordered_map>
#include <vector>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    unordered_map<string, int> map;
    for (string name : participant) {
        map[name]++;
    }
    for (string name : completion) {
        map[name]--;
        if (map[name] == 0) {
            map.erase(name);
        }
    }
    return map.begin()->first;
}
